### ISSUE TITLE
Lint files in parallel

### ISSUE THREAD

**Issue by anderseknert** [open-policy-agent/regal#113]
Testing Regal on a medium large policy repo (116 policy files) and unsurprisingly, it's slow — total time to lint clocked in at about 12 seconds. This is with all the rules enabled, mind you, but since we're likely making that the default, we should try and improve this before a first release. [Collecting and reporting metrics](https://github.com/StyraInc/regal/issues/26) will be of great help for further optimizations, but since most of the time is likely always spent evaluating Rego — and each input file is evaluated individually — it seems like an easy win to parallelize linting across all the input files.

An option to turn this off, or to configure the level of parallelism would be nice, but perhaps not needed for the first iteration.
